README.md

```diff
@@ -43,7 +43,17 @@ Want to get paid for your contributions to `one-app`?
 
 ### Quick Start
 
-**Clone and Install One App:**
+**Build a Module with [generator-one-app-module](https://github.com/americanexpress/one-app-cli/tree/master/packages/generator-one-app-module)**
+
+The easiest way to do this is via [npx](https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner) (comes with `npm` versions 5.2.0 and above). Run the following command in the directory you want your module to live:
+
+```bash
+npx -p yo -p @americanexpress/generator-one-app-module -- yo one-app-module
```

Comment:
what about changing `one-app-module` to `my-first-module` to make it clear that its the name

Reply:
`one-app-module` is the command to instantiate the generator, not the name of the module you're creating. 

Similarly to in v4 when the command was 
`npx -p yo -p generator-one-amex -- yo one-amex` (telling yeoman to go find the `one-amex` generator)

You declare the name of your module once the generator starts up and prompts you. I can see where this is unclear though, because the v4 generator allowed you to provide a subsequent arg after `yo one-amex` to declare the name of the module, i.e. `yo one-amex my-first-module`. The v5 generator doesn't have this option, you can only declare it in the prompt after start up.